public/editor/scripts/require.js

```diff
@@ -0,0 +1,16 @@
+function unloadProtection(e) {
```

Comment:
please keep this file named after what it does, not "require.js". `require.js` is the bundling software, this should probably be called "reload-protection.js" instead